hawkular-wildfly-agent-feature-pack/src/main/resources/configuration/standalone/template.xml

```diff
@@ -18,4 +18,85 @@
 
 -->
 <server xmlns="urn:jboss:domain:4.0">
+
+  <extensions>
+        <?EXTENSIONS?>
+  </extensions>
+
+  <management>
```

Comment:
@bobmcwhirter do these `subsystems.xml` and `template.xml` files have any significance for WF Swarm?